src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4849,4 +4849,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ElseClauseRequiredWithWhenClause" xml:space="preserve">
     <value>An 'else' clause is required when a 'when' clause is present.</value>
   </data>
-</root>
+  <data name="ERR_SemiOrLBraceOrArrowExpected" xml:space="preserve">
+    <value>{ or ; or =&gt; expected</value>
```

Comment:
Oh, sorry, I misunderstood (English isn't my native language). I understood it in the meaning of 'mentioned with single characters', ie. to quote => to mention ;-)